Scripts/Source/UD_CustomDevice_RenderScript.psc

```diff
@@ -3245,672 +3903,216 @@ float Function getMaxDurability()
     return UD_Health
 EndFunction
 
+;/  Function: getCondition
+    Returns:
+
+        Get absolute condition of device. Always in range from 0.0 to max device health
+/;
 float Function getCondition()
     return _total_durability_drain
 EndFunction
 
+;/  Function: getRelativeCondition
+    Returns:
+
+        Get relative condition of device. Always in range from 0.0 to max device health
```

Comment:
Isn't it returning value in range of 0.0..1.0 here?

Reply:
You are right. I copied it from getCondition and forget to change it. Fixed.